src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -603,5 +608,17 @@ private static string ConvertEndOfLineAndQuotationCharactersToEscapeForm(string
 
             return sb.ToString();
         }
+        private static bool ContainsAtSymbol(string value)=> value.Length > 0 && value[0] == '@';
+        private static string ProtectAtSymbol(string value)
```

Comment:
nit: this can be written as 

```C#
private static string ProtectAtSymbol(string value) => 
     ContainsAtSymbol(value) ? value : $"_{value}";
```